### DISCUSSION TITLE
초대코드 유효시간 부여 및 스터디 초대 기획 부분 변경

### DISCUSSION THREAD

**Discussion by rbgus2002** [rbgus2002/cactusdy#207] [Features]
이 부분은 길게 설명 안해도 잘 아실 거라 생각합니다 ㅎㅎ
생각해봤을 때 재우님이 말씀해주신 부분이 맞는 거 같아서 스터디 초대 코드에 유효시간을 설정하면 좋을 거 같아요.
다만 이 부분은 디자인&기획이 새롭게 들어가야하는 부분인데 기획만 해당 discussion에서 짜놓고 디자이너와 전체 디자인 개선 작업할 때 진행하면 좋을 거 같아요.

저는 아래처럼 생각하고 있습니다.
* 스터디 초대 화면 진입 시 (현재 스터디 상세페이지 내에 멤버 부분 `+` 버튼) 스터디 초대 코드 발급 후 **유효 시간 30분** 설정
* 스터디 초대 코드는 6자리로 현행 유지
* 스터디 초대 화면 너무 텅텅 빈 느낌이라 개선점 고민
* 카카오톡 초대 공유 시에 사용자에게 보여지는 문구 변경&사진 추가

**Comment by rbgus2002** [comment]
@Arkady1356

**Comment by Arkady1356** [comment]
1. 인정
2. 인정
3. 인정
4. 인정
여기서 스터디 설명 써도 좋겠다

**Reply by Arkady1356** [reply]
1년이 지난 지금, 다시 읽어보니 뭔 내용인지 기억 하나도 안 난다~
3번 4번을 지금이라도 다시 구체화 해도 좋을 것 같아요
